client/webserver/site/src/js/wallets.js

```diff
@@ -377,6 +377,7 @@ export default class WalletsPage extends BasePage {
     page.withdrawAmt.value = ''
     page.withdrawPW.value = ''
 
+    page.withdrawAvail.setAttribute('data-amount', Doc.formatConventionalUnits(wallet.balance.available, asset.info.unitinfo))
```

Comment:
Should use `page.withdrawAvail.dataset.amount = ...`, but let's not persist the info at all. The info gets stale and we're not watching for balance updates.